examples/config/2.2i_config.yml

```diff
@@ -66,21 +66,20 @@ TXTwoPoint:
     nbins: 20
     verbose: 0
 
-TXDiagnosticMaps:
-    chunk_rows: 100000  # Number of rows to process at once
-    sparse: True  # Generate sparse maps - faster if using small areas
-    snr_threshold: 10.0
-    snr_delta: 1.0
+TXMainMaps:
     pixelization: healpix
     nside: 1024
-    # pixelization: gnomonic
```

Comment:
just to clarify -- so we are fully healpix now, is that right? no for flat-sky possibilities?

Reply:
Thanks - this is a good point - I think it might be handy to at least make diagnostic and other maps in gnomonic, so I've pushed some changes to make that work properly.

l have (for now) left this out of TXTwoPointFourier stages - an informative error message is generated if you try.